common/beerocks/bwl/shared/netlink_socket.cpp

```diff
@@ -18,9 +18,7 @@ namespace bwl {
 netlink_socket::netlink_socket(int protocol)
     : m_nl_socket(nl_socket_alloc(), nl_socket_free), m_protocol(protocol)
 {
-    if (!m_nl_socket) {
-        LOG(ERROR) << "Failed to allocate netlink socket!";
-    }
+    LOG_IF(!m_nl_socket, ERROR) << "Failed to allocate netlink socket!";
```

Comment:
```suggestion
    LOG_IF(!m_nl_socket, FATAL) << "Failed to allocate netlink socket!";
```

This will allow aborting the application if `el::Loggers::addFlag(el::LoggingFlag::DisableApplicationAbortOnFatalLog);` is removed from our `beerocks_logging::apply_settings()` implementation.

@vitalybu why don't we want to abort on FATAL? Letting constructors fail on abort in such cases is much better than just having an error log and fail later.

Reply:
We don't always want to abort the application.
If for some reason prplMesh will run on a platform without access to netlink sockets, it could still function (to some extent).

Reply:
OK, valid point. Lets keep it as `ERROR` - but I think we should remove `el::Loggers::addFlag(el::LoggingFlag::DisableApplicationAbortOnFatalLog)` and start using `LOG(FATAL)` appropriately.